website/docs/plugin/framework/migrating/attributes-blocks/validators-predefined.mdx

```diff
@@ -0,0 +1,164 @@
+---
+page_title: 'Attribute predefined validators: Migrating from SDKv2 to the Framework'
+description: >-
+  Validations check required syntax, types, and acceptable values.
+  Migrate the predefined ConflictsWith, ExactlyOneOf, AtLeastOneOf and RequiredWith validators to the Framework.
+---
+
+# Validators - Predefined
+
+Attribute validators ensure that attributes do or do not contain specific values. You can use predefined validators for
+many use cases, or implement custom validators. Refer to [Schemas - Validators](/plugin/framework/schemas#validators) in
+the Framework documentation for details. Refer to the
+[Attributes - Custom Validators](/plugin/framework/migrating/validators-custom) page in this guide to learn how to
+implement custom validators.
+
+This page explains how to migrate a predefined validator from SDKv2 to the Framework.
+
+## SDKv2
+
+In SDKv2, the `ConflictsWith`, `ExactlyOneOf`, `AtLeastOneOf`, and `RequiredWith` fields on an attribute's
+`schema.Schema` struct perform predefined validations on the list of attributes set for these fields.
+
+```go
+func resourceExample() *schema.Resource {
+	return &schema.Resource{
+ 		...
+        Schema: map[string]*schema.Schema{
+            "attribute_example": {
+                ConflictsWith:  []string
+                ExactlyOneOf:	[]string
+                AtLeastOneOf:   []string
+                RequiredWith:   []string
+...
+```
+## Framework
+
+In the Framework, you implement either type of validation by setting the `Validators` field on the `tfsdk.Attribute`
+struct with types that satisfy the `tfsdk.AttributeValidator` interface. Validators that perform the same checks as the
+predefined validators in SDKv2 are
+[available for the Framework](https://github.com/hashicorp/terraform-plugin-framework-validators). If the predefined
+validators do not meet your needs, you must define
+[custom validators](/plugin/framework/migrating/validators-custom).
+
+```go
+func (d *resourceTypeExample) GetSchema(context.Context) (tfsdk.Schema, diag.Diagnostics) {
+	return tfsdk.Schema{
+        ...
+		Attributes: map[string]tfsdk.Attribute{
+            "attribute_example": {
+                Validators: []tfsdk.AttributeValidator{
+					schemavalidator.ConflictsWith(),
```

Comment:
Over in https://github.com/hashicorp/terraform-plugin-framework-validators/pull/60 we'll also add support for "config" validators for this type of functionality that may be a little bit more intuitive for provider developers since you declare them outside a specific attribute. I think showing the 1:1 is nice here, but it might also be worth calling those out in the future since I think they'll be preferable.

Reply:
I've called out the "config" validators and added a generic example for usage with resources.

Reply:
You rock, thank you